shared/protos/garbage/garbage.proto

```diff
@@ -0,0 +1,234 @@
+syntax = "proto3";
+
+package google.garbage.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/resource.proto";
+import "google/longrunning/operations.proto";
+import "google/protobuf/timestamp.proto";
+
+option (google.api.client_package) = {
+  title: "Testing Garbage"
+  namespace: ["Google", "Cloud"]
+  version: "v1"
+};
+
+// Endless trash
+service GarbageService {
+  // The address where this API is typically served.
+  option (google.api.default_host) = "garbage.googleapis.com";
+
+  // The OAuth scopes needed to make requests to this API.
+  option (google.api.oauth_scopes) = "https://cloud.google.com/auth/cloud-platform,"
+                                     "https://cloud.google.com/auth/garbage";
+
+  rpc GetSimpleGarbage(SimpleGarbage) returns (SimpleGarbage) {
+    option (google.api.http) = {
+      post: "/v1/simple_garbage:get"
+      body: "*"
+    };
+  }
+  rpc GetSpecificGarbage(SpecificGarbage) returns (SpecificGarbage) {
+    option (google.api.http) = {
+      post: "/v1/specific_garbage:get"
+      body: "*"
+    };
+  }
+  rpc GetRepeatedGarbage(RepeatedGarbage) returns (RepeatedGarbage) {
+    option (google.api.http) = {
+      post: "/v1/repeated_garbage:get"
+      body: "*"
+    };
+  }
+  rpc LongRunningGarbage(LongRunningGarbageRequest) returns (google.longrunning.Operation) {
+    option (google.api.http) = {
+      post: "/v1/garbage:lro"
+      body: "*"
+    };
+    // Once the operation is complete, the ultimate result
+    // will be a LongRunningGarbageResponse.
+    option (google.longrunning.operation_info) = {
+      response_type: "google.garbage.v1.LongRunningGarbageResponse"
+      metadata_type: "google.garbage.v1.LongRunningGarbageMetadata"
+    };
+  }
+
+  // This method split the given content into words and will pass each word back
```

Comment:
Why do these methods have the same comments as showcase (copy/paste)? Some comments are fine but we probably shouldn't use the same one here.

Reply:
Removed.